relay-filter/src/transaction_name.rs

```diff
@@ -18,6 +18,10 @@ pub fn should_filter(
     event: &Event,
     config: &IgnoreTransactionsFilterConfig,
 ) -> Result<(), FilterStatKey> {
+    if !config.is_enabled || config.patterns.is_empty() {
```

Comment:
This is the same check as the `is_empty` method, right?

```suggestion
    if config.is_empty() {
```

Reply:
of course, you are right ! Thanks.